components/BottomAppBar/examples/BottomAppBarTypicalUseExample.m

```diff
@@ -45,9 +45,10 @@ - (void)commonBottomBarTypicalUseExampleInit {
   [self.bottomBarView.floatingButton setImage:addImage forState:UIControlStateNormal];
 
   // Theme the floating button.
-  MDCBasicColorScheme *colorScheme =
-      [[MDCBasicColorScheme alloc] initWithPrimaryColor:[UIColor whiteColor]];
-  [MDCButtonColorThemer applyColorScheme:colorScheme toButton:self.bottomBarView.floatingButton];
+  MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
+  colorScheme.secondaryColor = [UIColor whiteColor];
```

Comment:
Do we need to change the secondary color?

Reply:
FAB is using secondary not primary.

Reply:
I can get rid of it here but I think it's more reasonable to do this in a separate PR, since this is just replacing an old API with the new one. What do you think?

Reply:
Done.